xbmc/platform/win32/filesystem/Win32File.cpp

```diff
@@ -177,9 +176,9 @@ ssize_t CWin32File::Read(void* lpBuf, size_t uiBufSize)
 
   if (uiBufSize == 0)
   { // allow "test" read with zero size
-    XUTILS::auto_buffer dummyBuf(255);
+    std::vector<char> dummyBuf(255);
```

Comment:
A `const char dummyBuf = 0;` would be sufficient.

Reply:
not sure how i missed this one, thanks